agent_pool_integration_test.go

```diff
@@ -68,6 +68,21 @@ func TestAgentPoolsList(t *testing.T) {
 		assert.Nil(t, pools)
 		assert.EqualError(t, err, ErrInvalidOrg.Error())
 	})
+
+	t.Run("with query options", func(t *testing.T) {
+
```

Comment:
Ah, I believe you mentioned this query param was sitting behind a feature flag... in which case `skipIfBeta(t)` test helper will need to be added to this subtest otherwise the test will fail in CI... I don't think tflocal has the flag enabled?

Reply:
@sebasslash sorry, took a vacation and let this review sit for a bit. The flag is now on in production, and it looks like we are seeing the tests pass!